cfme/tests/cli/test_appliance_console.py

```diff
@@ -730,9 +745,8 @@ def test_appliance_console_ha_setup_dc():
     pass
 
 
-@pytest.mark.manual
 @pytest.mark.tier(2)
-def test_appliance_console_restart():
+def test_appliance_console_restart(appliance):
```

Comment:
Agreed with @john-dupuy here - this needs to be using a temp_appliance fixture, as an issue with the restart could cause cascading test failures/skips.